src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,61 @@
-export const Tabs = () => {};
+import React from 'react';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+interface TabsProps {
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+  tabs: Tab[],
+}
+
+export const Tabs: React.FC<TabsProps> = ({
+  tabs, selectedTabId, onTabSelected,
+}) => {
+  function findTab(tabId: string): Tab {
+    return tabs.find((tab) => tab.id === tabId) || tabs[0];
+  }
```

Comment:
assign it to a variable directly instead of creating a function

Reply:
A function is created because it is reused at lines 24 and 28

Reply:
Unless you mean to use an arrow function instead, which would function virtually the same

Reply:
Approach with a separate function worked perfectly; I do not know why I did not think of that at all